applications/ShallowWaterApplication/custom_processes/depth_integration_process.cpp

```diff
@@ -82,7 +82,7 @@ void DepthIntegrationProcess<TDim>::Execute()
             results.resize(max_results);
```

Comment:
`.resize(max_results, false)`

Reply:
It seems that the suggested modification leads to errors during the compilation